sdk/eventhub/Azure.Messaging.EventHubs/MIGRATIONGUIDE.md

```diff
@@ -0,0 +1,396 @@
+# Migration Guide (EventHubs v4 to v5)
+
+This document is intended for users that are familiar with v4 of the .NET SDK for Event Hubs library ([`Microsoft.Azure.EventHubs`](https://www.nuget.org/packages/Microsoft.Azure.EventHubs/) & [`Microsoft.Azure.EventHubs.Processor`](https://www.nuget.org/packages/Microsoft.Azure.EventHubs.Processor/)) and wish 
+to migrate their application to v5 of the same library.
```

Comment:
Since in .Net, the new libraries are not new versions of the same library unlike Python & JS, we might want to reword this part.

How about the below?

```suggestion
to migrate their application to the newer Azure.Messaging.EventHubs and Azure.Messaging.EventHubs.Processor libraries.
```